recipe/conda_build_config.yaml

```diff
@@ -1,3 +1,6 @@
 mpi:
```

Comment:
Look [here](https://github.com/conda-forge/mpi4py-feedstock/blob/main/recipe/conda_build_config.yaml). You need something like

```yaml
mpi:
  - mpich    # [not win]
  - openmpi  # [not win]
  - msmpi    # [win]
```